lambdas/credential-subject/tests/credential-subject-handler.test.ts

```diff
@@ -51,13 +51,13 @@ describe("credential-subject-handler.ts", () => {
     expect(credentialSubject).toEqual(expectedCredentialSubjectWithBirthDates);
   });
 
-  it.each([{}, undefined, null])(
+  it.each([{ govJourneyId: mockUserInfoEventItem }])(
```

Comment:
Is this really equivalent? the lambda was designed not to error and to always return {} object if the input is unexpected maybe I have missed something?